packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -65,26 +66,40 @@ class BaseFileLink(BaseModel):
         description="Entity tag that uniquely represents the file. The method to generate the tag is not specified (black box).",
         alias="eTag",
     )
+    label: str = Field(
+        None,
```

Comment:
no it is not optional. it has a default value in the base class, that *must* be available for a datcore link, and is auto-filled in case of a simcore link.

Reply:
done